notebooks/uncertainty_aware_models/solutions/uncertainty_quantification_with_sngp_bert.ipynb

```diff
@@ -0,0 +1,1092 @@
+{
```

Comment:
<p><code>bert_ood_labels</code> -&gt; <code>ood_labels</code> , and reuse in sngp model since the label is the same? Later we see some undefined <code>ood_labels</code> errors. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/262/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>Done! (by fixing the missing ood_labels later)</p>